pkg/process/process.go

```diff
@@ -186,3 +187,11 @@ func (p *Process) IsStopped() bool {
 	defer p.lock.RUnlock()
 	return p.State == StateStopped || p.State == StateError
 }
+
+func IsEngineProcess(name string) bool {
+	// engine process name example: pvc-5a8ee916-5989-46c6-bafc-ddbf7c802499-e-0
+	nameSlices := strings.Split(name, "-")
+	processType := nameSlices[len(nameSlices)-2]
+
+	return processType == "e"
```

Comment:
Use regex instead. Also, probably can add to common lib if appropriate, as it is used in your another longhorn manager PR. 

Suggest adding a doc comment for this function to better explain what the arg means.

Reply:
PR: https://github.com/longhorn/go-common-libs/pull/25